src/index.js

```diff
@@ -483,10 +504,10 @@ function ncc (
         
         ({ code, map } = {
           code: result.code,
-          map: sourceMap ? JSON.parse(result.map) : undefined
+          map: map ? JSON.parse(result.map) : undefined
```

Comment:
Do we need to update `map` above so that it includes mjs too?

https://github.com/vercel/ncc/blob/2ba1c2290d8fead03cba66088961ab308fc698f6/src/index.js#L436-L439

Reply:
The source maps seem to be working to me. Would rather leave this alone as a legacy fix - it may be possible to remove it now possibly.

Reply:
I've posted https://github.com/webpack/webpack/issues/13740 to follow up on this.

Reply:
Double-checked and this is definitely only needed for the `.cjs` cases as a webpack hack there. `.mjs` still works fine.